frontend/styles/var/colors.ts

```diff
@@ -13,10 +13,11 @@ export const COLORS = {
   "ui-fleet-black-50": "#8B8FA2",
   "ui-fleet-black-33": "#B3B6C1",
   "ui-fleet-black-25": "#C5C7D1",
-  "ui-fleet-black-15": "#E2E4EA",
+  "ui-fleet-black-10": "#E2E4EA",
   "ui-off-white": "#F9FAFC",
   "ui-blue-hover": "#5D5AE7",
   "ui-blue-pressed": "#4B4AB4",
+  "ui-blue-focused-outline": "#D9D9FE",
```

Comment:
this is the same as line 22, do we want to have one name per color?

Reply:
nope. didnt notice we had it already. thanks for that. removed it

can I get rereview when you have a sec?